plugins/src/main/java/opengrok/auth/plugin/UserWhiteListPlugin.java

```diff
@@ -44,18 +44,31 @@
     private static final String className = UserWhiteListPlugin.class.getName();
     private static final Logger LOGGER = Logger.getLogger(className);
 
+    // configuration parameters
     static final String FILE_PARAM = "file";
+    static final String FIELD_PARAM = "fieldName";
+
+    // valid values for the FIELD_NAME parameter
+    static final String USERNAME_FIELD = "username";
+    static final String ID_FIELD = "id";
 
     private final Set<String> whitelist = new TreeSet<>();
+    private String fieldName;
```

Comment:
can be final

Reply:
It cannot. The value is assigned in `load()`.